CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,489 @@
+  "schemaVersion": "1.0.0",
+  "certificationLevel": 1,
+  "certificateIssuer": {
+    "name": "Audit House LLC",
+    "logo": "https://www.example.com/media/logo.svg"
+    "social": {
+        "contact": "contact@example.com",
+        "link": "https://example.com",
+        "twitter": "twiterHandle",
+        "github": "githubHandle",
+        "website": "https://www.example.com"
+    }
+  },
+  "report": {
+    "reportURLs": [
+        "https://example.io/certificate.pdf",
+        "ipfs://bafybeiemxfal3jsgpdr4cjr3oz3evfyavhwq"
+        ],
+    "reportHash": "c6bb42780a9c57a54220c856c1e947539bd15eeddfcbf3c0ddd6230e53db5fdd"
+  },
+  "summary": "This is the summary of the report."
+  "disclaimer": "This is the legal disclaimer from the report",
+  "scripts": [
+        {
+          "smartContractInfo": {
+            "era": "basho",
+            "compiler": "plutusTx",
+            "compilerVersion": "1.3.0",
+            "optimizer": "plutonomy",
+            "optimizerVersion": "v0.20220721",
+            "progLang": "plutus-v2",
+            "repository": "https://github.com/DAppDev/TestDApp/"
+          },
+          "fullScriptHash": "711dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+          "scriptHash": "1dcb4e80d7cd0ed384da5375661cb1e074e7feebd73eea236cd68192",
+          "contractAddress": "addr1wywukn5q6lxsa5uymffh2esuk8s8fel7a0tna63rdntgrysv0f3ms"
+        },
+        {
+          "smartContractInfo": {
+            "era": "basho",
+            "compiler": "plutusTx",
+            "compilerVersion": "1.3.0",
+            "optimizer": "plutonomy",
+            "optimizerVersion": "v0.20220721",
+            "progLang": "plutus-v2",
+            "repository": "https://github.com/DAppDev/TestDApp/"
+          },
+          "fullScriptHash": "384da5375661cb1e07713eea236cd681921dcb4e80d7cd0ed4e7feebd7",
+          "scriptHash": "6cd68191dcb4e80d7c5661cb1e074e7feebd73eea232d0ed384da537",
+          "contractAddress": "addr1wywukn5q6lrdntgrysv0f7a0tna63ymffh23ms5uxsaesuk8s8fel"
+        }
+  ]
+}
+```
+
+The metadata should be discoverable by all certification stakeholders, including end-users, DApp developers, and ecosystem components, such as light wallets and DApp stores. Information should be indexable by certification issuer, DApp developer, DApp and DApp version.
+
+It should be possible for there to be multiple versions of metadata published by the same certification issuer for the same version of a DApp, particularly when a (minor) update of the evidence is necessary. In such a case it should be possible to identify the most recent version of the report for that DApp version. It will also be the case that the same DApp may have certification metadata provided by multiple different certification issuers.
+
+It should be possible for wallets to identify to users the certification status of a DApp when they are signing a transaction that is being submitted to a deployed DApp.
```

Comment:
[RSoulatIOHK](https://github.com/RSoulatIOHK)'s answer comment answer [here](https://github.com/cardano-foundation/CIPs/pull/499#issuecomment-1584798813).

Thanks that makes sense to me - IMO it would be nice to have this more explicit answer in the CIP 😎